Documentation/Settings.md

```diff
@@ -0,0 +1,302 @@
+<!-- This is an automatically generated file. DO NOT EDIT. Edit gen_menu_docs.py instead -->
+
+# IronOS Settings Menu
+
+The below breaks down the menu's and what each setting means.
+    
+## Menu Categories
+
+In the menu there are a few main categories that are used to keep the list manageable.
+
+### Category: Power settings
+
+Menu for settings related to power. Main settings to do with the input voltage.
+
+### Category: Soldering settings
+
+Setttings for soldering mode, such as tip temperature, boost temps, sleep temps.
```

Comment:
1. In Set***t***tings there is one extra **t**.
2. In **`Soldering settings`** there is no dedicated *tip temperature-setting*.
Besides *boost temp* it regulates the behavior while powering up, which increments is to be used if the buttons are pressed short/long and whether the buttons could be locked.
3. Sleep temp is handled in **`Sleep mode`**.